server/internal/deployments/validations.go

```diff
@@ -0,0 +1,82 @@
+package deployments
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/speakeasy-api/gram/server/internal/constants"
+	"github.com/speakeasy-api/gram/server/internal/functions"
+)
+
+const (
+	maxDisplayNameLength = 128
+	maxSlugLength        = 128
+)
```

Comment:
These are not meant as true limits but generous enough upper bounds. we'll need to figure out a better way to share the true limits between our code and the Goa design layer. This is however a decent temporary measure.

Reply:
@disintegrator Want to drop the db length check and only do it here?
https://github.com/speakeasy-api/gram/blob/47da0e45ec63fe5ede061f2423558a8cd492851a/server/database/schema.sql#L235